packages/flutter/lib/src/services/autofill.dart

```diff
@@ -801,7 +801,8 @@ class _AutofillScopeTextInputConfiguration extends TextInputConfiguration {
   _AutofillScopeTextInputConfiguration({
     required this.allConfigurations,
     required TextInputConfiguration currentClientConfiguration,
-  }) : super(inputType: currentClientConfiguration.inputType,
+  }) : super(viewId: currentClientConfiguration.viewId,
+         inputType: currentClientConfiguration.inputType,
```

Comment:
formatting nit: I think this should be
```suggestion
  }) : super(
         viewId: currentClientConfiguration.viewId,
         inputType: currentClientConfiguration.inputType,
```

(I know this was busted before...)

Reply:
Done